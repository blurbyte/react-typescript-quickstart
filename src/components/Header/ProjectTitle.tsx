import styled from '@styles/styledComponents';
import { Colors } from '@styles/colors';

const ProjectTitle = styled.p`
  text-transform: uppercase;
  cursor: default;
  flex-shrink: 0;
  border-left: 1px solid ${Colors.gray};
  margin: 0;
  margin-left: 1.4rem;
  padding: 1rem 1.4rem;
  user-select: none;
`;

export default ProjectTitle;
