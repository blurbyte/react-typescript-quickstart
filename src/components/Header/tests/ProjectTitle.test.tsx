import * as React from 'react';
import { shallow } from 'enzyme';

import ProjectTitle from '../ProjectTitle';

describe('ProjectTitle', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<ProjectTitle />);
    expect(wrapper).toMatchSnapshot();
  });
});
