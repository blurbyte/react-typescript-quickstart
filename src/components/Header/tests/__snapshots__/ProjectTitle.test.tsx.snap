// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProjectTitle renders correctly 1`] = `
.c0 {
  text-transform: uppercase;
  cursor: default;
  -webkit-flex-shrink: 0;
  -ms-flex-negative: 0;
  flex-shrink: 0;
  border-left: 1px solid #908e89;
  margin: 0;
  margin-left: 1.4rem;
  padding: 1rem 1.4rem;
  -webkit-user-select: none;
  -moz-user-select: none;
  -ms-user-select: none;
  user-select: none;
}

<p
  className="c0"
/>
`;
